soccer/NewRefereeModule.cpp

```diff
@@ -328,6 +334,18 @@ void NewRefereeModule::updateGameState(bool blueTeam) {
             break;
         case Command::GOAL_BLUE:
             break;
+        case Command::BALL_PLACEMENT_YELLOW:
```

Comment:
The refbox cant send that command without already being in the stop state but if it were to be set to ignore the rules this could break it so I will add that.